setup/bundles.rst

```diff
@@ -173,7 +173,7 @@ Before diving into the specifics of the most common edge cases, the general
 recommendation is to **not rely on the Symfony Kernel version** to decide which
 code to use::
 
-    if (Kernel::VERSION_ID <= 20800) {
+    if (Kernel::VERSION_ID < 20800) {
```

Comment:
TL;DR : there is one exceptional case when it is ok to use the kernel constant comparison : when you deal with the http kernel component itself. That's why I think it is important to get this sample piece of code right.